Assets/AppsFlyer/IAppsFlyerNativeBridge.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace AppsFlyerSDK
+{
+
+    public interface IAppsFlyerNativeBridge
```

Comment:
Maybe we can check the nested interface, for example:

```
IAppsFlyerNativeBridgeAndroid -> IAppsFlyerNativeBridgeBase

IAppsFlyerNativeBridgeiOS -> IAppsFlyerNativeBridgeBase
```
So we can get rid of redundant methods